failed to resolve: could not find `LazyTupleUnion` in `strategy`

When I derive Arbitrary on a very simple enum, I get the titular error.

source: 
```
#[cfg_attr(test, derive(Arbitrary))]
pub enum BraceKind {
    Parentheses,
    Square,
    Curly,
}
```

compile error:
```
error[E0433]: failed to resolve: could not find `LazyTupleUnion` in `strategy`
 --> src\lexer.rs:7:25
  |
7 | #[cfg_attr(test, derive(Arbitrary))]
  |                         ^^^^^^^^^ could not find `LazyTupleUnion` in `strategy`
  |
  = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)
error[E0412]: cannot find type `LazyTupleUnion` in module `_proptest::strategy`
   --> src\lexer.rs:7:25
    |
7   | #[cfg_attr(test, derive(Arbitrary))]
    |                         ^^^^^^^^^ help: a struct with a similar name exists: `TupleUnion`
    | 
   ::: C:\Users\Izaak Weiss\.cargo\registry\src\github.com-1ecc6299db9ec823\proptest-0.10.1\src\strategy\unions.rs:340:1
    |
340 | pub struct TupleUnion<T>(T);
    | ---------------------------- similarly named struct `TupleUnion` defined here
    |
    = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)
Some errors have detailed explanations: E0412, E0433.
For more information about an error, try `rustc --explain E0412`.
```

cargo expand output:
```
    #[allow(non_upper_case_globals)]
    const _IMPL_ARBITRARY_FOR_BraceKind: () = {
        extern crate proptest as _proptest;
        impl _proptest::arbitrary::Arbitrary for BraceKind {
            type Parameters = ();
            type Strategy = _proptest::strategy::LazyTupleUnion<(
                (u32, ::std::sync::Arc<fn() -> Self>),
                (u32, ::std::sync::Arc<fn() -> Self>),
                (u32, ::std::sync::Arc<fn() -> Self>),
            )>;
            fn arbitrary_with(_top: Self::Parameters) -> Self::Strategy {
                {
                    let () = _top;
                    _proptest::strategy::LazyTupleUnion::new((
                        (1u32, ::std::sync::Arc::new(|| BraceKind::Parentheses {})),
                        (1u32, ::std::sync::Arc::new(|| BraceKind::Square {})),
                        (1u32, ::std::sync::Arc::new(|| BraceKind::Curly {})),
                    ))
                }
            }
        }
    };
```

--- a/book/src/proptest-derive/getting-started.md
+++ b/book/src/proptest-derive/getting-started.md
@@ -5,7 +5,7 @@
 To the `[dev-dependencies]` section of your `Cargo.toml`, add
 
 ```toml
-proptest-derive = "0.1.0"
+proptest-derive = "0.2.0"
 ```
 
 In a Rust 2015 crate, you must add

